docs/writingrules.rst

```diff
@@ -738,6 +738,15 @@ the C API.
             $text_string
     }
 
+Unreferenced strings
+--------------------
+
+YARA 4.4.0 allows for unreferenced strings in the condition. If a string
+identifier starts with an ``_`` then it does not have to be referenced in the
+condition. Any other string must be referenced in the condition. This is useful
+if you want to search for particular strings and handle them in a custom
+callback but don't really need them for your condition logic.
+
```

Comment:
An example would be nice, is not clear if the underscore must be before or after the `$`.